### PR TITLE
Invalid example url

### PR DESCRIPTION
**Pull request by zuston**
**What this PR does / why we need it**:
Invalid example url

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/metricscollector/v1beta1/tfevent-metricscollector/tfevent_loader.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 # TFEventFileParser parses tfevent files and returns an ObservationLog of the metrics specified.
 # When the event file is under a directory(e.g. test dir), please specify "{{dirname}}/{{metrics name}}"
-# For example, in the TensorFlow official tutorial for mnist with summary (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/tutorials/mnist/mnist_with_summaries.py),
+# For example, in the kubeflow tf-operator tutorial for mnist with summary (https://github.com/kubeflow/tf-operator/blob/master/examples/v1/mnist_with_summaries/mnist_with_summaries.py),
 # the "accuracy" metric is saved under "train" and "test" directories. So in Katib, please specify name of metrics as "train/accuracy" and "test/accuracy".
```
Comment (andreyvelich) [inline pkg/metricscollector/v1beta1/tfevent-metricscollector/tfevent_loader.py:12]: I believe this part is also incorrect.

```suggestion
# the "accuracy" metric is saved under "train" and "test" directories. So in the Metrics Collector specification, please specify name of "train" or "test" directory. Check TFJob example for more information: https://github.com/kubeflow/katib/blob/master/examples/v1beta1/tfjob-example.yaml#L16-L22
```

**Review by andreyvelich — COMMENTED** [review]
Thank you for your contribution @zuston!
I left one comment, I think we should also modify the example part.

**Comment by andreyvelich** [comment]
As well, please sign the CLA.

**Comment by zuston** [comment]
> Thank you for your contribution @zuston!
> I left one comment, I think we should also modify the example part.

Thanks reply. Fix again.

**Comment by andreyvelich** [comment]
> > Thank you for your contribution @zuston!
> > I left one comment, I think we should also modify the example part.
> 
> Thanks reply. Fix again.

Thank you! You need to follow https://cla.developers.google.com/ to sign the CLA.

**Comment by zuston** [comment]
I signed it! @googlebot

**Review by andreyvelich — APPROVED** [review]
Thank you for doing this @zuston!
/lgtm
/approve

**Pull request merged**